File: dp_primer/dp_primer_recursive_formula_boss.cs
// 実行時間 10ms
using static System.Console;
using static System.Linq.Enumerable;

internal class Program
{
    static void Main()
    {
        int Q = int.Parse( ReadLine() );
        int[] a = new int[100];

        SetOut( new System.IO.StreamWriter( OpenStandardOutput() ) { AutoFlush = false } );
        a[0] = 1;
        a[1] = 1;
        foreach( int i in Range( 2, 98 ) ) {
            a[i] = a[i - 2] + a[i - 1];
        }
        foreach( int _ in Range( 0, Q ) ) {
            WriteLine( a[int.Parse( ReadLine() ) - 1] );
        }
        Out.Flush();
        return;
    }
}

File: dp_primer/dp_primer_recursive_formula_boss.py
# 実行時間 20ms
q = int(input())
a = [1, 1]

for i in range(2, 100):
    a.append(a[i - 2] + a[i - 1])
for _ in [0] * q:
    print(a[int(input()) - 1])
